lib/resque/failure/extra.rb

```diff
@@ -114,6 +114,22 @@ def requeue_and_remove(index, queue=nil)
         mark_for_remove(index)
       end
 
+      # remove_older_than deletes all jobs older than the days provided
+      def remove_older_than(days)
```

Comment:
Do we want to restrict usage to such a long interval?
The refreshed UI takes care of some accidents by defaulting to 30 days.